### PR TITLE
genshi: Initial add of genshi templating library

### PR DESCRIPTION
**Pull request by jcftang**
Signed-off-by: Jimmy Tang <jcftang@gmail.com>

### CODE REVIEW

File: pkgs/genshi.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+extends: [setuptools_package]
+
+dependencies:
+  build: []
+  run: []
+
+sources:
+ - url: https://pypi.python.org/packages/source/G/Genshi/Genshi-0.7.zip
+   key: zip:ptgbosdhel3aawcwn3pqmd6iy2xqvqkv
```
Comment (certik) [inline pkgs/genshi.yaml]: switch url and key...

**Comment by ahmadia** [comment]
Looks good.

**Comment by jcftang** [comment]
@ahmadia @certik cheers! merging this

**Comment by certik** [comment]
+1

**Pull request merged**